programmers/수박수박수박수박수.js

```diff
@@ -0,0 +1,6 @@
+function solution(n) {
+    if(n % 2 === 0) {
+        return('���'.repeat(n / 2));
+    } 
+    return('���'.repeat(n / 2) + '��');
+}
```

Comment:
```suggestion
function solution(n) {
   return n % 2 ? '수박'.repeat(n / 2) + '수' : '수박'.repeat(n / 2);
}
```
이거 한국어 깨지는거 상당히 곤란한데... 내일 고칩시다
그리고 if문 사용할때 if와 괄호 시작 한칸  띄어주세요.

만든 코드에서 어제오늘 공부한 삼항연산자로 변경할 수 있군요
if문을 사용하면 삼항연산자로 한 번 바꿔보는 연습도 좋을듯요.

```js
function solution(n) {
   return "수박".repeat(n).substr(0, n)
}
```

이건 내일 숙제에요 ~